src/store/flow/block.ts

```diff
@@ -80,16 +80,31 @@ export const mutations: MutationTree<IFlowsState> = {
     const block = findBlockOnActiveFlowWith(blockId, state as unknown as IContext)
     findBlockExitWith(exitId, block).semantic_label = value
   },
-  block_addExit(state, {blockId, exit}: {blockId: string, exit: IBlockExit}) {
+  block_addExit(state, {blockId, exit, shouldAppend}: {blockId: string, exit: IBlockExit, shouldAppend?: boolean}) {
     const block = findBlockOnActiveFlowWith(blockId, state as unknown as IContext)
-    // insert before default exit
-    block.exits.splice(block.exits.length - 1, 0, exit)
+
+    if (shouldAppend) {
+      block.exits.push(exit)
+    } else {
+      block.exits.splice(block.exits.length - 1, 0, exit)
```

Comment:
could we please add a short comment what this is doing? is this inserting before the last element (default exit?)